src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -756,6 +756,11 @@ public ImmutableArray<T> RemoveRange(int index, int length)
             Requires.Range(index >= 0 && index < self.Length, "index");
             Requires.Range(length >= 0 && index + length <= self.Length, "length");
 
+            if (length == 0)
+            {
+                return self;
```

Comment:
Probably no change, since `self.Length` would have thrown. I just thought I'd check... is there a test for the case of calling this on an uninitialized instance?